simples3.go

```diff
@@ -173,6 +175,15 @@ func (s3 *S3) SetEndpoint(uri string) *S3 {
 	return s3
 }
 
+// SetToken can be used to the set a token for
+// using an alternate instance compatible with the s3 API.
```

Comment:
Hey,

I think we should change this doc line to mention Temporary Security Credentials

Reply:
Like you have mentioned above (IAM role or AWS STS)